packages/citizen-sdk/src/constants.ts

```diff
@@ -27,6 +27,25 @@ export const Envs: Record<string, Record<string, string>> = {
   },
 }
 
+/**
```

Comment:
If these are just placeholders and not working this should be clarified in a readme and should be configurable options for developers to pass down
(we are not hosting a farcaster app, developers integrating the identitysdk/claim-sdk might)